src/Game.java

```diff
@@ -0,0 +1,115 @@
+import java.util.*;
+
+public class Game {
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+    private static final int MAX_ERRORS = 8;
+
+    private Scanner scanner = new Scanner(System.in);
+
+    private String wordToFind;
+    private char[] guessedWord;
+    private int mistakesMade;
+    private ArrayList<String> words = new ArrayList<>();
+
+    Keywords keywords = new Keywords();
+    WordsFromFile wordsFromFile = new WordsFromFile();
+
+    public void hangman() {
+        int option = -1;
+        while (option != EXIT) {
+
+            printOption();
+
+            option = scanner.nextInt();
+            scanner.nextLine();
+
+            switch (option) {
+                case EXIT:
+                    System.out.println("Bye bye!");
+                    scanner.close();
+                    break;
+                case PLAY:
+                    play();
+                    break;
+                default:
+                    System.out.println("Wybrałeś nieprawidłową opcję");
```

Comment:
What happens if i will choose wrong option? Lets say 3.

Reply:
A message is displayed stating that the wrong option has been selected and we ask the user to select the option again. You haven't tested my app? :D

Reply:
Nope, if I were testing every PR I am looking at I wouldn't do anything else